src/visual.ts

```diff
@@ -483,25 +490,39 @@ module powerbi.extensibility.visual {
                 TextMeasurementService.svgEllipsis);
 
             if (this.settings.misc.isAnimated && this.settings.misc.isStopped) {
-                this.main.selectAll(LineDotChart.Line.selector).selectAll(LineDotChart.dotPointsText).remove();
+                this.main.selectAll(LineDotChart.Line.selectorName).selectAll(LineDotChart.dotPointsText).remove();
                 this.line.selectAll(LineDotChart.textSelector).remove();
 
                 return;
             }
 
+            if (this.settings.axisOptions.show === true) {
+                this.SetAxisColor(this.settings.axisOptions.color);
+            } else {
+                this.SetAxisColor(LineDotChart.axesDefaultColor);
+            }
             let linePathSelection: d3.selection.Update<LineDotPoint[]> = this.line
                 .selectAll(LineDotChart.dotPathText)
                 .data([this.data.dotPoints]);
-
-            this.drawLine(linePathSelection);
-            this.drawClipPath(linePathSelection);
-
             linePathSelection
                 .exit().remove();
-
+            this.drawLine(linePathSelection);
+            this.drawClipPath(linePathSelection);
             this.drawDots();
         }
 
+        public SetAxisColor(color: string): void {
```

Comment:
Name of the method must be in camel case. In other words, it should be `setAxisColor`